src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/model/Sentence.kt

```diff
@@ -54,11 +54,13 @@ data class Sentence(
       )
   }
 
+  @JsonIgnore
```

Comment:
added these annotations so that these behavioural attributes arent stores in the json representation of the Sentence object (in the DB) - will be discussing with the TA whether we should avoid using JSON columns to keep the code neater